server/public/styles.css

```diff
@@ -0,0 +1,71 @@
+.page {
+  height: 100vh;
+  width: 100%;
+  background: rgb(147, 32, 224);
+  background: linear-gradient(0deg, rgba(147, 32, 224, 1) 0%, rgba(12, 0, 83, 1) 100%);
+  display: flex;
+  justify-content: center;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.form-container {
+  background: rgb(12, 0, 83);
+  border-radius: 10px;
+  width: 345px;
+  padding: 20px 50px;
+}
+
+.form-title {
+  margin: 0 auto;
+  margin-bottom: 0.7rem;
+  font-weight: 400;
+}
+
+.column {
+  flex-direction: column;
+}
+
+.center-content {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
```

Comment:
I don't think this is necessary. Your form is over-using flexbox. divs will stack on top of each other automatically. If you need to squeeze box contents inward, add left and right padding. Try only using flexbox for the space-between in your buttons.